backend/app/assets/stylesheets/spree/backend/sections/_stock_management.scss

```diff
@@ -7,17 +7,17 @@
   }
 
   > tbody {
-    > tr {
-      > td {
-        background-color: $color-tbl-odd;
-      }
-      &.even td {
+    > tr.stock-item-edit-row {
+      td {
         background-color: $color-tbl-even;
       }
-      &:first-child {
-        td {
-          border-top: none;
-        }
+      &:nth-child(2n) td {
```

Comment:
You can express this as `nth-child(even)` if you want it to read a little nicer.